pkg/config/template.go

```diff
@@ -253,6 +253,10 @@ ctr_stop_timeout = {{ .CtrStopTimeout }}
 # It requires manage_ns_lifecycle to be true.
 drop_infra_ctr = {{ .DropInfraCtr }}
 
+# infra_ctr_cpuset determines what CPUs will be used to run infra containers.
+# You can use linux CPU list format to specify desired CPUs.
```

Comment:
I also think it would be worth mentioning that this option needs to be specified with kubelet's `--reserved-cpus` for a container to be truly guaranteed a cpu

Reply:
Makes a lot of sense, done